SP-3226: Updates for DP2 release, PPDB and image services (#67)
lsst/rtn-011: access.tex

Rubin data products will be served to the community from the US Data Access Center (US DAC) hosted in the Google Cloud.\footnote{\url{data.lsst.cloud}}.
A number of Rubin Independent Data Access Centers (IDAC) are also under construction to provide additional user computing resources to LSST users around the globe \citedsp{RTN-003}.

> Rubin employs a \emph{hybrid cloud} architecture in which science data products are stored at the USDF and made accessible to users via the US DAC in Google  Cloud.

Reviewer: could \cite{2024SPIE13101E..2BO} for hybrid if you wish

> The deep coadd data products are an exception: because there is one coadd per  sky region per band (a relatively small and slowly-growing dataset), coadd images  are hosted directly in Google Cloud, independently of USDF storage.

Reviewer 2: @frossie I'm assuming that we are using this particular phrasing (and not "cache" framing) on purpose?

Author: We can pick this up in the next issue

> The deep coadd data products are an exception: because there is one coadd per  sky region per band (a relatively small and slowly-growing dataset), coadd images  are hosted directly in Google Cloud, independently of USDF storage.

Reviewer 3: ```suggestion
The deep coadd data products are an exception: because there is one coadd per sky region per band (a relatively small and slowly-growing dataset), coadd images are hosted directly in Google Cloud, independently of USDF storage.
```

> This architectural distinction has implications for the DP2 delivery timeline: products served directly from Google Cloud (deep coadds and catalogs) can be released as Early DP2 ahead of products that require real-time data transfer from the USDF (visit-level images), which depend on sufficient hybrid cloud network  bandwidth.

Reviewer 4: ```suggestion
This architectural distinction has implications for the DP2 delivery timeline: products served directly from Google Cloud (deep coadds and catalogs) can be released as Early DP2 ahead of products that require real-time data transfer from the USDF (visit-level images), which depend on full hybrid cloud US DAC readiness.
```

Reviewer 4: IIUC the network bandwidth is just the current bottleneck, and we have a list of other technical challenges to address before going live with the hybrid US DAC. Suggest a more general statement here that will not go out of date so quickly? (The gateway server upgrade work is nearly done.)

Author: It is *the* problem. But I can change the text

> See \S~\ref{ssec:dp2} for the phased DP2 release schedule and the engineering  work underway to address the bandwidth constraint.

Reviewer 4: ```suggestion
See \S~\ref{ssec:dp2} for the phased DP2 release schedule and the engineering  work underway.
```

\item DataLink annotations in TAP query results for access to light curves and other related information;
\item ObsCore data model for image metadata;
\item IVOA-compatible SIAv2 and ObsTAP image metadata query services;
\item IVOA SODA service for cutouts from individual single-epoch images and coadded image tiles\footnote{At this time, the cutout service can only process requests for one cutout at a time, meaning to create and retrieve 10 cutouts will require 10 independent synchronous calls to the cutout service.

> A bulk cutout service is under development and expected to be available by DR1};

Reviewer 2: ```suggestion
A bulk cutout service is under development and expected to be available by DR1.};
```

(footnote should be a complete sentence)
